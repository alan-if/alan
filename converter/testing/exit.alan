location l
  exit w to l.
  exit w, w to l.
  exit w to l.
end location.

location n
  exit w,w to l.
end location n.

location m
  exit w to l.
  exit w to l.
end location.

start at l.
