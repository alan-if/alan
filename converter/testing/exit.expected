########## exit ##########


The l Isa location
  
  Exit w To l.
  
  Exit w, w To l.
  
  Exit w To l.
  
End The l.

The n Isa location
  
  Exit w, w To l.
  
End The n.

The m Isa location
  
  Exit w To l.
  
  Exit w To l.
  
End The m.

Start At l.
  
